### PR TITLE
update ci to use erlang foundation version

### PR DESCRIPTION
**Pull request by princemaple**
image

### CODE REVIEW

**Comment by princemaple** [comment]
same as https://github.com/elixir-lsp/elixir_sense/pull/123

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ jobs:
       MIX_ENV: test
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-elixir@v1
+      - uses: erlef/setup-elixir@v1@v1
```
Comment (lukaszsamson) [inline .github/workflows/ci.yml:44]:
```suggestion
      - uses: erlef/setup-elixir@v1
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -66,7 +66,7 @@ jobs:
             otp: 23.2.x
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-elixir@v1
+      - uses: erlef/setup-elixir@v1@v1
```
Comment (lukaszsamson) [inline .github/workflows/ci.yml:69]:
```suggestion
      - uses: erlef/setup-elixir@v1
```

File: .github/workflows/release-asset.yml [hunk 3]
```diff
@@ -45,7 +45,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Set up Elixir
-        uses: actions/setup-elixir@v1
+        uses: erlef/setup-elixir@v1@v1
```
Comment (lukaszsamson) [inline .github/workflows/release-asset.yml:48]:
```suggestion
        uses: erlef/setup-elixir@v1
```

**Comment by princemaple** [comment]
😂

**Comment by lukaszsamson** [comment]
Thanks

**Pull request merged**